apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/condition/request/ReadConditionRequest.java

```diff
@@ -0,0 +1,18 @@
+package gov.cdc.nbs.questionbank.condition.request;
+
+import lombok.Data;
+
+@Data
+public class ReadConditionRequest {
+
+    private String searchText;
+
+    private String filterField;
+
+    private String filterValue;
```

Comment:
can they only send 1 filter at a time or should this be a list of filter objects?